### PR TITLE
Wym/ Wdym added to filter

### PR DESCRIPTION
**Pull request by MajManatee**
# Document the changes in your pull request
I said it the other day and people got mad.
Never again.

# Changelog

:cl:  
tweak: Wym / Wdym added to filter.
tweak: tf added to the filter.
/:cl:

### CODE REVIEW

**Comment by ToasterBiome** [comment]
wym

**Comment by solwashere** [comment]
it was me I got mad

**Comment by TheRyeGuyWhoWillNowDie** [comment]
Zero thumbs-up emojis. Why is this even being considered?

**Comment by Moltijoe** [comment]
Gonna conflict with the pls/plz one

**Comment by MajManatee** [comment]
yeah i figured, but for time reasons ill fix it after

**Comment by ToasterBiome** [comment]
Cornflakes

**Review by ToasterBiome — DISMISSED** [review]
spellcheck is the incorrect changelog to use here

File: config/minor_filter.txt [hunk 1]
```diff
@@ -40,5 +40,8 @@ hbu=how about you
 \bffs\b=for fuck's sake
 \bur\b=your
 btw=by the way
+wym=what do you mean
+wdym=what do you mean
 pls=please
-plz=please
\ No newline at end of file
+plz=please
+tf=the fuck
```
Comment (Moltijoe) [inline config/minor_filter.txt]:
```suggestion
\btf\b=the fuck
```

File: config/minor_filter.txt [hunk 2]
```diff
@@ -40,5 +40,8 @@ hbu=how about you
 \bffs\b=for fuck's sake
 \bur\b=your
 btw=by the way
+wym=what do you mean
```
Comment (Moltijoe) [inline config/minor_filter.txt]:
```suggestion
\bwym\b=what do you mean
```

**Review by Moltijoe — CHANGES_REQUESTED** [review]
probably isn't needed for wdym because i don't think any word uses those letters in that order

**Review by Moltijoe — APPROVED** [review]

**Review by ToasterBiome — APPROVED** [review]

**Pull request merged**